### DISCUSSION TITLE
Listings endpoint does not return all elements

### DISCUSSION THREAD

**Discussion by timokuelbel** [etsy/open-api#958] [Q&A]
When I look in my etsyshop I see 236 active items. But if I call the listings api as follows:
https://openapi.etsy.com/v3/application/shops/26673399/listings/active?limit=100
I see as count only 212 items is this a bug, or why does this amount differ. Even if I iterate step by step with an offset, I get exactly 212 items.
The endpoint https://openapi.etsy.com/v3/application/shops/26673399/listings?limit=100&offset=0 delivers the right elements.

**Comment by turboturtleD** [comment]
I think there is already an existing discussion on this (with some insight).

https://github.com/etsy/open-api/discussions/873

**Reply by Gareth064** [reply]
I was trying to find that discussion... and I had bloody commented in it too. I find GitHubs search (for issues and discussions) to be a bit flakey

**Comment by mikeytt** [comment]
I'm seeing this issue as well. I use the findAllActiveListingsByShop to get the ID number of  new listing o it can be entered into a filemaker database for easy access to the listing in the future. With v2 there was no problem, the new listing would appear immediately when queried. With v3 the new listing does not immediately appear in the query (the count stays the same as before the new listing was made) . After some random time the listing then does appear and the count is then correct.

**Comment by turboturtleD** [comment]
Not immediately appearing is like another issue all together that will probably not be resolved any time soon (or ever), likely to do with their core API/indexing.

You can likely call it directly though by listingID which I imagine is in the returned response from when you created the new listing.  Might not be ideal, but therein likely lies your workaround if this is critical.

**Reply by mikeytt** [reply]
Shame that the index no longer updates instantly like under v2. Unfortunately I am not creating the listing using the api, just using the api to get the ID after it has been created. 

However there does appear to be another workaround by using  **getListingsByShopSectionId**  instead of  **findAllActiveListingsByShop**. The SectionID index seems to be updated instantly.

**Comment by timokuelbel** [comment]
Thanks for all answers
